package.json

```diff
@@ -6,9 +6,9 @@
   "scripts": {
     "pretest": "NODE_ENV=test sequelize db:migrate && NODE_ENV=test sequelize db:seed:all && NODE_ENV=test npm run create_admin",
     "test": "NODE_ENV=test nyc mocha test/**.spec.js --require babel-polyfill --require babel-core/register --exit",
-    "posttest": "NODE_ENV=test sequelize db:migrate:undo:all",
+    "posttests": "NODE_ENV=test sequelize db:migrate:undo:all",
```

Comment:
I would like to know why you altered the above script's name

Reply:
That was changed while testing to check if the notification was working. I'll change it back.